### ISSUE TITLE
Fixed Assets

### ISSUE THREAD

**Issue by nthock** [waynerobinson/xeroizer#361]
Hi,

Is there a method in Xeroizer to get the fixed assets data in Xero?

Thanks,
Thiam Hock

**Comment by CloCkWeRX** [comment]
https://developer.xero.com/documentation/assets-api/overview ? Not yet, but pull requests welcome.

To do it, you'd probably want to copy the approach taken on the payroll branch - inject the right endpoint, put models in a new module, etc.

**Comment by nthock** [comment]
Thanks! I will look at the payroll branch to see how it works. This is an awesome gem that simplify my development process. Hope to contribute once I am done with my development work.

**Comment by nthock** [comment]
Hi @CloCkWeRX,

I tried to make use of the code in Xeroizer for me to connect to Assets API endpoint by defining a separate model class. But it doesn't seems to work. I got a 403 response code (forbidden).

However, when I changed the `url` method to return any of the Accounting API endpoint, the code is able to get the data from Xero.

I think I still have not completely understand how Xeroizer and Xero API works in general. Can I connect to the Assets API using the same public application that I have created using Xeroizer?

This is my code for extracting Assets API for your reference.
https://gist.github.com/nthock/eec59b2043498ff984f8a0f57244ddb8
